### PR TITLE
NATIVE-579: Feature Flag: Make Icons in the intro slides configurable

### PR DESCRIPTION
**Pull request by maxammann**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **NATIVE**.

### CODE REVIEW

**Review by Schedulaar — APPROVED** [review]
To me it would make more sense to have a folder structure like
```
build-configs
  * integreat
      ** assets
          *** asset1.png
      ** theme.js
      ** config.js
  * malte
    ...
```
to have everything of one build-config at the same place...
Maybe that's just preference though...

File: .idea/runConfigurations/Start_packager_for_Integreat.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <component name="ProjectRunConfigurationManager">
```
Comment (klinzo) [inline .idea/runConfigurations/Start_packager_for_Integreat.xml:1]: This starts the integreat-test-cms packager. Not sure if the naming is good therefore.

File: src/modules/app/constants/buildConfig.js [hunk 2]
```diff
@@ -2,24 +2,54 @@
...
+  intro: {
+    events: string,
+    language: string,
+    offers: string,
+    search: string
+  }
+|}
+
+export const buildConfigIconSet = (): IconSetType => {
+  const iconSetString = buildConfig().assets
```
Comment (klinzo) [inline src/modules/app/constants/buildConfig.js:31]: Either use the new term `assets` completely or stick to `iconSet`.

**Review by klinzo — APPROVED** [review]

File: .idea/runConfigurations/Start_packager_for_Integreat.xml (hunk 1 above)
Comment (maxammann) [reply .idea/runConfigurations/Start_packager_for_Integreat.xml]: I renamed them

File: src/modules/app/constants/buildConfig.js (hunk 2 above)
Comment (maxammann) [reply src/modules/app/constants/buildConfig.js]: done

**Comment by maxammann** [comment]
@Schedulaar done

**Review by klinzo — APPROVED** [review]

**Pull request merged**